Resources/skeleton/crud/actions/edit.php.twig

```diff
@@ -17,7 +17,7 @@
     {
 {% block method_body %}
         $deleteForm = $this->createDeleteForm(${{ entity_singularized }});
-        $editForm = $this->createForm(new {{ entity_class }}Type(), ${{ entity_singularized }});
+        $editForm = $this->createForm({{ entity_class }}Type::class, ${{ entity_singularized }});
```

Comment:
Actually, this does not work in PHP versions before 5.5. To be on the safe side you should probably rather use the FCQN instead.